pandas/core/arrays/datetimes.py

```diff
@@ -1202,7 +1202,8 @@ def month_name(self, locale=None) -> npt.NDArray[np.object_]:
         ----------
         locale : str, optional
             Locale determining the language in which to return the month name.
-            Default is English locale.
+            Default is English locale (``'en_US.utf8'``). Use the command
+            ``locale -a`` on your terminal to look for your locale language code.
```

Comment:
```suggestion
            ``locale -a`` on your terminal on Unix systems to find your locale language code.
```

Could you change the other usages of this below too?

Reply:
Even better! I'll change that on the day_name docstring as well.